scripts/pre-deploy-check.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Pre-deploy check script for DJ Zen Eyer
+
+echo "🚀 Iniciando verificações pré-deploy..."
+
+# 1. Verificar erros de sintaxe JSX/TSX
+echo "🔍 Verificando lint..."
+npm run lint
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)